### ISSUE TITLE
Failed to precompile in julia 1.11.0

### ISSUE THREAD

**Issue by PhyX-Meow** [compintell/Mooncake.jl#287]
```
Precompiling Mooncake...
Info Given Mooncake was explicitly requested, output will be shown live
ERROR: LoadError: UndefVarError: `arraylen` not defined in `Core.Intrinsics`
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base ./Base.jl:42
 [2] top-level scope
   @ ~/.julia/packages/Mooncake/GMFJI/src/rrules/builtins.jl:76
 [3] include(mod::Module, _path::String)
   @ Base ./Base.jl:557
 [4] include(x::String)
   @ Mooncake ~/.julia/packages/Mooncake/GMFJI/src/Mooncake.jl:1
 [5] top-level scope
   @ ~/.julia/packages/Mooncake/GMFJI/src/Mooncake.jl:80
 [6] include
   @ ./Base.jl:557 [inlined]
 [7] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
   @ Base ./loading.jl:2806
 [8] top-level scope
   @ stdin:4
in expression starting at /home/wuli/.julia/packages/Mooncake/GMFJI/src/rrules/builtins.jl:36
in expression starting at /home/wuli/.julia/packages/Mooncake/GMFJI/src/Mooncake.jl:1
in expression starting at stdin:4
  ✗ Mooncake
  ✗ Mooncake → MooncakeSpecialFunctionsExt
  0 dependencies successfully precompiled in 17 seconds. 85 already precompiled.
```

**Comment by willtebbutt** [comment]
Hi @PhyX-Meow and thanks for opening this issue. We're aware of this, and are working to fix it in the next 24 hours. Please see #180 for links to various discussions.

**Comment by willtebbutt** [comment]
This should be resolved by #288 . Please re-open if version 0.4.11 does not resolve this problem.
